LocalSettings.php

```diff
@@ -2015,6 +2015,10 @@
                                  'en', 
                           ],
 		],
+		'+r4356thwiki' => [
+			'scratchwiki',
+			'snapwiki'
+		]
```

Comment:
Needs a comma

Reply:
While that is not really necessary, I will add it anyway. Though the example snippet in CONTRIBUTING.md did not mention it.

Reply:
Done.

Reply:
Thanks, @Pix1234. I feel badly for missing the comma. @R4356th, when you say the comma is "not really necessary," do you mean the configuration would've still worked without it? If _so_, I did not know that.

Reply:
@dmehus, no, I thought Zppix was talking about line 2020, which was optional and not widely used (but used in this repository probably because it makes adding more items to arrays easier later). It would not have worked without the comma on line 2021.